.circleci/config.yml

```diff
@@ -201,6 +201,47 @@ jobs:
                 name: Notify Slack on failure
                 when: on_fail
         working_directory: ~/kubernetes-monitor
+    deploy_sysdig_integration_cluster:
```

Comment:
deploy_to_sysdig_integration_cluster?

Reply:
just makes it sound like you are deploying the whole sysdig cluster 🤷

Reply:
Updated!

Reply:
:chefkiss: